### PR TITLE
feat(curriculum): add English curriculum block 7.2 Reading and understanding code documentation

### PR DESCRIPTION
**Pull request by miyaliu666**
Checklist:

- [x] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org).
- [x] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request).
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

### CODE REVIEW

File: curriculum/challenges/english/21-a2-english-for-developers/learn-how-to-read-and-understand-code-documentation/65fd5d1a6cb7f10f77a06163.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+`To be faced with` means to encounter or confront something, usually a challenge. For example, `She was faced with a difficult decision` meaning being in a situation where she needs to deal with something. 
+
+A `table of contents` is a list at the beginning of a document, showing its structure. Like in a book, `The table of contents lists all the chapters.`
+
+# --fillInTheBlank--
+
+## --sentence--
+
+`Don't _, When you're _ with extensive documentation, the first thing you should do is check for a _ of contents or an index.`
```
Comment (moT01) [inline curriculum/challenges/english/21-a2-english-for-developers/learn-how-to-read-and-understand-code-documentation/65fd5d1a6cb7f10f77a06163.md:23]:
```suggestion
`Don't _. When you're _ with extensive documentation, the first thing you should do is check for a _ of contents or an index.`
```

**Review by moT01 — COMMENTED** [review]
Looks good, found one minor typo I think.

**Review by moT01 — APPROVED** [review]

File: curriculum/challenges/english/21-a2-english-for-developers/learn-how-to-read-and-understand-code-documentation/66001d8b878f5141214f2d77.md [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+
+It means not being clear about something. Tom is unclear about how to understand the parameters and options.
+
+---
+
+`parameters`
+
+### --feedback--
+
+THey are special settings in functions or commands. Tom needs help understanding these.
```
Comment (naomi-lgbt) [inline curriculum/challenges/english/21-a2-english-for-developers/learn-how-to-read-and-understand-code-documentation/66001d8b878f5141214f2d77.md:37]:
```suggestion
They are placeholder variables in the function.
```

Not sure about the suggestion, but what we have now doesn't feel right at all.

**Review by naomi-lgbt — CHANGES_REQUESTED** [review]

File: curriculum/challenges/english/21-a2-english-for-developers/learn-how-to-read-and-understand-code-documentation/66001d8b878f5141214f2d77.md (hunk 2 above)
Comment (miyaliu666) [reply curriculum/challenges/english/21-a2-english-for-developers/learn-how-to-read-and-understand-code-documentation/66001d8b878f5141214f2d77.md:37]: @naomi-lgbt Thanks for catching this error. I searched other tasks to check the descriptions for "parameter", and I'll update task-28.

**Review by naomi-lgbt — APPROVED** [review]

**Pull request merged**